index.js

```diff
@@ -86,6 +74,7 @@ module.exports = function(clientId, clientSecret, config) {
 	var getTeamId = function(cb) {
 		getRequest('https://api.github.com/orgs/'+ config.organization + '/teams', function(err, res, body) {
 			if (err) return cb(err);
+			if (res.statusCode === 403) return cb(false);
```

Comment:
check the http status when get the organisation teams, if it is 403 which means you do not belong to this team.